pkg/virt-controller/watch/migration.go

```diff
@@ -598,6 +600,33 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 	return nil
 }
 
+func setTargetPodSELinuxLevel(pod *k8sv1.Pod, vmiSeContext string, logger *log.FilteredLogger) error {
+	// The target pod may share resources with the sources pod (RWX disks for example)
+	// Therefore, it needs to share the same SELinux categories to inherit the same permissions
+	// Note: there is a small probablility that the target pod will share the same categories as another pod on its node.
+	//   It is a slight security concern, but not as bad as removing categories on all shared objects for the duration of the migration.
+	if vmiSeContext == "" {
```

Comment:
Seems we do `"selinuxContext": "none",`

Reply:
Really nice catch, thank you! Should be fixed now.

Reply:
I agree @brianmcarey